db/seeds.rb

```diff
@@ -3,9 +3,30 @@
 Book.delete_all
 Author.delete_all
 List.delete_all
+Genre.delete_all
+
+Genre.transaction do
+  names = []
+  10.times { |n| names.push("#{Faker::Hacker.ingverb} books") }
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.